be/src/common/tracer.h

```diff
@@ -0,0 +1,43 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2022-present, StarRocks Limited.
+
+#pragma once
+
+#include "opentelemetry/trace/provider.h"
+#include "opentelemetry/sdk/trace/simple_processor.h"
+#include "opentelemetry/sdk/trace/tracer_provider.h"
+
+namespace starrocks {
+
+using Span = opentelemetry::nostd::shared_ptr<opentelemetry::trace::Span>;
+using SpanContext = opentelemetry::trace::SpanContext;
+
+class Tracer {
```

Comment:
maybe add an example about how to use this class in class comments

Reply:
> maybe add an example about how to use this class in class comments

Thanks a lot for the review, I will add test cases and usage examples later.